requirements.txt

```diff
@@ -65,3 +65,9 @@ gevent==1.4.0
 # Uncomment the requirement for ldap3 if using ldap.
 # It is not included by default because of the GPL license conflict.
 # ldap3==2.2.4
+
+# PyMongo and Athena dependencies are needed for some of the unit tests:
+# (this is not perfect and we should resolve this in a different way)
+pymongo[srv,tls]==3.9.0
+botocore>=1.12.241,<1.13.0
+PyAthena>=1.5.0
```

Comment:
Why move them here?

Reply:
Because the dependencies were needed for athena and mongo data sources. https://percy.io/Redash/redash/builds/2935315
Frontend test uses Dockerfile(not Dockerfile.dev) since it no need dev dependencies for server side.

Reply:
It's ok -- the Percy snapshots will be updated not to include them.